core/unit_test/tools/TestWithoutInitializing.cpp

```diff
@@ -142,13 +142,10 @@ TEST(kokkosp, create_mirror_view_and_copy) {
         device_view = mirror_device;
```

Comment:
Not directly part of this PR's goal, but should the above `return;` statements be changed to `GTEST_SKIP();`?